assets/messages.yml

```diff
@@ -489,23 +573,27 @@ messages:
         For any new or ongoing hashtag issues, please create a new ticket for
         any gamertags still affected and use the label {{[chat-filter|https://bugs.mojang.com/issues/?jql=labels+%3D+chat-filter]}}.
       fillname: []
-  i-am-a-bot:
+  hserr:
     - project:
-        - bds
         - mc
-        - mcapi
-        - mcd
-        - mce
         - mcl
-        - mcpe
+      name: Attach hserr_pid file
+      shortcut: hserr
+      message: |-
+        Is there a file with a name starting with {{hs_err_pid}} in your {{[https://minecrafthopper.net/help/guides/finding-minecraft-data-folder/|.minecraft]}} folder? If so, please attach it here.
```

Comment:
We should also tell the user to make their report private, see [MC-207889](https://bugs.mojang.com/browse/MC-207889)